demo/scripts/build-kind-image.sh

```diff
@@ -48,4 +48,4 @@ KIND_K8S_DIR="${TMP_DIR}/kubernetes-${KIND_K8S_TAG}"
 git clone --depth 1 --branch ${KIND_K8S_TAG} ${KIND_K8S_REPO} ${KIND_K8S_DIR}
 
 # Build the kind base image
-kind build node-image --base-image "${KIND_IMAGE_BASE}" --image "${KIND_IMAGE}" "${KIND_K8S_DIR}"
+kind build node-image --image "${KIND_IMAGE}" "${KIND_K8S_DIR}"
```

Comment:
As a matter of interest, is supporting the `KIND_IMAGE_BASE` something we want to remove? Or are we not too concerned with this at this stage?

Reply:
OK. That's fine. Looking at the code I see that there's still a way to build a custom k8s version in. I was looking at this since the images on Dockerhub seem to lag a little.

Reply:
Yes, you can still build an image with an arbitrary k8s version in it (i.e. against a `node` image`). I just removed the code for building the `node` image itself against a custom `base` image.